conf/application.conf

```diff
@@ -1,49 +1,124 @@
+common.newCategoryExternalId = "1"
+common.newCategoryExternalId = ${?NEW_CATEGORY}
+
 
-# Basic auth
+# SUGGESTIONS
 # ~~~~~
-application.auth.realm = "Sunrise Demo"
-application.auth.realm = ${?BASIC_AUTH_REALM}
 
-#application.auth.credentials = "username:password" # uncomment to enable HTTP basic access authentication with the given credentials
-application.auth.credentials = ${?BASIC_AUTH_CREDENTIALS}
+# The categories where to obtain the products shown in the home page suggestions. Enter a list of the category external IDs.
+home.suggestions.externalId = ["29", "37", "22", "28"]
+home.suggestions.externalId = ${?HOME_SUGGESTIONS_CATEGORIES}
+
+# The amount of products displayed in the home page suggestions.
+home.suggestions.count = 4
+home.suggestions.count = ${?HOME_SUGGESTIONS_SIZE}
+
+# The amount of products displayed for a product's suggestions.
+pdp.productSuggestions.count = 4
+pdp.productSuggestions.count = ${?PRODUCT_SUGGESTIONS_SIZE}
 
 
-# The application configuration
+
+# PRODUCT SEARCH
 # ~~~~~
-application.metrics.enabled = true
-application.setup.enabled = true
 
-#application.countries = ["DE", "US", "GB", "AT", "CH"] # uncomment to override countries from CTP project, default first
+# Field in the URL query string containing the searched term.
+pop.fulltext.key = "q"
+pop.fulltext.key = ${?SEARCH_FIELD}
+
+# Field in the URL query string containing the sorting criteria.
+pop.sort.key = "sort"
+pop.sort.key = ${?SORT_FIELD}
+
+# Field in the URL query string containing the amount of products per page.
+pop.pageSize.key = "display"
+pop.pageSize.key = ${?PRODUCTS_PER_PAGE_FIELD}
+
+# Available options to select the amount of products per page.
+pop.pageSize.options = [24, 48]
+pop.pageSize.options = ${?PRODUCTS_PER_PAGE_OPTIONS}
+
+# The selected amount of products per page by default.
+pop.pageSize.default = 24
+pop.pageSize.default = ${?PRODUCTS_PER_PAGE_DEFAULT}
+
+# The maximum amount of pages to be displayed in the pagination, which the customer can use to navigate directly.
+pop.pagination.displayedPages = 6
+pop.pagination.displayedPages = ${?PAGINATION_DISPLAYED_PAGES}
+
+# Order to use for the size facet results.
+pop.facet.size = ["one Size","XXS","XS","S","M","L","XL","XXL","XXXL","34","34.5","35","35.5","36","36.5","37","37.5",
+  "38","38.5","39","39.5","40","40.5","41","41.5","42","42.5","43","43.5","44","44.5","45","45.5","46","46.5","47",
+  "47.5","70","75","80","85","90","95","100","105","110","115","120"]
+
+
+# CHECKOUT
+# ~~~~~
+
+checkout.allowedTitles = [
+  {"message":"form.personTitles.mr", "value":"mr"},
+  {"message":"form.personTitles.mrs", "value":"mrs"},
+  {"message":"form.personTitles.ms", "value":"ms"},
+  {"message":"form.personTitles.dr", "value":"dr"}
+]
+
+checkout.customerServiceNumber = "+49 899982996-0"
+
+
+# LOCALIZATION
+# ~~~~~
+
+# LEAVE COMMENTED IF YOU WANT THE APPLICATION TO USE THE COUNTRIES DEFINED IN YOUR PROJECT!
```

Comment:
I think project is to generic, how about
`LEAVE COMMENTED IF YOU WANT THE APPLICATION TO USE THE COUNTRIES DEFINED IN YOUR COMMERCETOOLS PROJECT!`?